source/nodejs/package.json

```diff
@@ -6,7 +6,8 @@
 		"start:designer": "lerna run start --scope=adaptivecards-designer --stream",
 		"start:designer-app": "lerna run start --scope=adaptivecards-designer-app --stream",
 		"start:all": "lerna run --parallel watch && lerna run --parallel start",
-		"audit-all": "lerna-audit"
+		"audit-all": "lerna-audit",
+		"prepare": "cd ../.. && husky install source/nodejs/.husky"
```

Comment:
It the format takes more than half an minutes or have a lot of outputs, I would like to opt in or at least put it into push hook.

Reply:
formatting `source/nodejs/adaptivecards/**/*.ts` takes `6.45s` on my machine

Reply:
for what it's worth, linting and formatting is [recommended](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) in `pre-commit`